functions/verify-phone.ts

```diff
@@ -5,7 +5,6 @@ import fetch from 'node-fetch';
 export const handler: Handler = async (event: HandlerEvent) => {
   const payload = JSON.parse(event.body);
   const { access_key, number, country_code, format } = payload || {};
-  console.log(number);
```

Comment:
Could you please merge or rebase main branch? this one already fixed

Reply:
I did merge from upstream and it was still there for me?

Reply:
yep, I've added some code in order to we can easily test PR and now you 9 commits behind 
![image](https://user-images.githubusercontent.com/16488867/120015296-d794b900-bfeb-11eb-812f-c4394c4c2253.png)

Could I ask you one more thing: close this PR and create new one. Test action will create only on fresh PR